---
bool is_desc_loaded
